pipeline/metadata/6.0.yaml

```diff
@@ -214,3 +214,33 @@
     - ibmc
     - rgw
     - stage-1
+
+- name: "Testing RBD basic regression scenarios"
```

Comment:
We would need stage information. You could check or reach out to the concerned team. If we don't have stage... the suites would not be picked up in the Sanity pipeline.

Reply:
@psathyan Done